forge/test/mlir/test_ops.py

```diff
@@ -583,7 +583,45 @@ def forward(self, x, y):
 
     co_out = [co.to("cpu") for co in co_out]
     fw_out = [fw_out] if isinstance(fw_out, torch.Tensor) else fw_out
-    assert all([compare_with_golden_pcc(golden=fo, calculated=co, pcc=0.99) for fo, co in zip(fw_out, co_out)])
+    assert all([compare_with_golden(golden=fo, calculated=co) for fo, co in zip(fw_out, co_out)])
+
+
+@pytest.mark.parametrize(
+    "shape",
+    [
+        (1, 128, 28, 28),
+        (1, 64, 28, 28),
+        (1, 256, 28, 28),
+        (1, 128, 14, 14),
+        (1, 128, 56, 56),
+        (1, 32, 64, 64),
+        (1, 512, 7, 7),
+        (1, 32, 32, 32),
+    ],
+)
+@pytest.mark.push
+def test_equal(shape):
+    class equal(nn.Module):
```

Comment:
Can we use the [CapWords](https://peps.python.org/pep-0008/#class-names) convention for naming classes? 

I see this is not the case for all tests, but should be